app/controllers/public/sessions/list.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+import { action, computed  } from '@ember/object';
+
+export default class extends Controller {
+  isTrackVisible = false;
+  @action
+  toggleTrack() {
+    this.toggleProperty('isTrackVisible');
+  }
+
+  @computed('model.event.startsAt', 'model.event.endsAt')
+  get allDates() {
+    let arr = new Array();
```

Comment:
Codacy found an issue: ['arr' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6034570)